esp/src/src-react/hooks/workunit.ts

```diff
@@ -317,3 +317,21 @@ export function useWorkunitHelpers(wuid: string): [HelperRow[], () => void] {
 
     return [helpers, incCounter];
 }
+
+export function useGlobalWorkunitNotes(): [WUDetails.Note[]] {
```

Comment:
minor: Slightly confusing to have the name Workunit in the function name.  useGlobalNotes would be clearer and more accurate (since the errors etc are not necessarily stored in the global workunit)..